public/app/plugins/datasource/prometheus/querybuilder/parsing.test.ts

```diff
@@ -216,15 +249,23 @@ describe('buildVisualQueryFromString', () => {
             params: [2],
           },
         ],
-      })
-    );
+      },
+    });
   });
 
   it('parses binary operation with 2 queries', () => {
     expect(
       buildVisualQueryFromString('avg_over_time(http_requests_total{instance="foo"}[$__interval]) / sum(logins_count)')
-    ).toEqual(
-      noErrors({
+    ).toEqual({
```

Comment:
So it's interesting that all of the new failure cases on this branch are related to the `__interval`, or `__rate_interval` variables...